src/helpers/screenings.js

```diff
@@ -3,13 +3,52 @@ import fetch from "node-fetch";
 const API = "https://plankton-app-xhkom.ondigitalocean.app/api/screenings";
 
 export async function movieScreenings(id) {
-    const res = await fetch(`${API}?filters[movie]=${id}`);
-    const payload = await res.json();
-    return payload.data;
+  const res = await fetch(`${API}?filters[movie]=${id}`);
+  const payload = await res.json();
+  return payload.data;
 }
 
 export async function getAllScreenings() {
-    const res = await fetch(API);
-    const payload = await res.json();
-    return payload.data;
+  const res = await fetch(API);
+  const payload = await res.json();
+  return payload.data;
+}
+
+export async function screeningsStartpage() {
+  const apiRes = await fetch(API + "?populate=movie");
```

Comment:
This will only return the first 25 screenings in the database, which are likely very old. Because all of the screenings returned are old, they will be discarded by the logic below, and no screenings show up on the home page.